packages/app/src/app/pages/Sandbox/Editor/Header/ForkButton/ForkButton.tsx

```diff
@@ -143,27 +156,40 @@ export const ForkButton: React.FC<ForkButtonProps> = props => {
 
   return (
     <Stack>
-      <Button
-        onClick={() => props.forkClicked()}
-        loading={state.editor.isForkingSandbox}
-        variant={props.variant}
-        css={{
-          width: 'calc(100% - 26px)',
-          borderTopRightRadius: 0,
-          borderBottomRightRadius: 0,
-        }}
-      >
-        <ForkIcon css={css({ height: 3, marginRight: 1 })} /> Fork
-      </Button>
-      <Menu>
-        <Menu.Button
+      {state.activeWorkspaceAuthorization === 'READ' ? null : (
```

Comment:
Do we completely hide the button? Or disable it? It might be good to disable the button and have a tooltip that explains why it's disabled, so it doesn't look like a bug.

Reply:
We completely hide it right now

Reply:
Added a tooltip 👍

Reply:
Super nice!! One thing I haven't made up my mind about is whether we should hide the fork button, or disable it if you can't make forks. What do you think is best? Whatever you think is best, this is ready to be merged!